### PR TITLE
User can choose UI language

### PR DESCRIPTION
**Pull request by DavDan1**
PT: https://www.pivotaltracker.com/story/show/176030915

### User Story
```
As a user
In order to view the application in my preferred language
I would like to be able choose between English and Swedish
```

### Changes proposed
- Installs and Configures i18n
- adds dropdown menu to choose between English and Swedish languages
- adds two files (en.js and se.js) to hold translations
- adds translation to all UI elements and popups
______
### Packages installed
- i18next
______
### What we learned
- Working with i18next
- Some swedish 

https://user-images.githubusercontent.com/57390886/119500691-5c6fa080-bd68-11eb-99db-aa4802461992.mp4

### CODE REVIEW

File: src/languages/en.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    //----BACKYARD DASHBOARD----
+    backyardDashboardHeaderLocationOn: 'Backyard Conspiracies from',
+    backyardDashboardHeaderLocationOff: 'Allow your location!',
+    backyardDashboardTitle: 'Title',
+    backyardDashboardTheme: 'Theme',
+    backyardDashboardWrittenBy: 'Written By',
+    backyardDashboardCreatedOn: 'Created On',
+    backyardDashboardAction: 'Action',
+    backyardDashboardView: 'View',
+    //----LOGIN----
```
Comment (LucasKnudsen) [inline src/languages/en.js:26]: Is it possible to nest stuff like this? 
```
backyardDashboard: {
  headerLocationOn: 'blbl'
  ...
}
```

**Review by LucasKnudsen — APPROVED** [review]
Very cool!

File: src/languages/en.js (hunk 1 above)
Comment (DavDan1) [reply src/languages/en.js:26]: I don't think so, but we can doublecheck 👯‍♂️

Comment (offcast-dev) [reply src/languages/en.js:26]: Double checked, you can do it. If you do this nesting you need to use following syntax: `t('backyardDashboard.headerLocationOn')`
We don't think that it worth refactoring our code, it is simple but time consuming. But definitely good thing to know in the future.

Comment (LucasKnudsen) [reply src/languages/en.js:26]: Ye I agree, not worth it at this point probably - but ye, will make it more readable and a little less writing :P

File: cypress/integration/userCanChangeUILanguage.feature.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  context('choosing Swedish language', () => {
+    it('is expected to translate UI into Swedish', () => {
+      cy.get('[data-cy=language-dropdown]').click();
+      cy.get('.visible.menu.transition').contains('se').click();
+      cy.get('[data-cy=home-tab]').should('contain', 'Hem');
+      cy.get('[data-cy=backyard-tab]').should('contain', 'Bakgard');
```
Comment (lukeperrin10) [inline cypress/integration/userCanChangeUILanguage.feature.js:11]: Bakgard here, shouldn't it be bakgård or does cypress now pick up the å character?

**Review by lukeperrin10 — APPROVED** [review]
Looks good

**Review by arunbhalli — APPROVED** [review]

File: cypress/integration/userCanChangeUILanguage.feature.js (hunk 2 above)
Comment (DavDan1) [reply cypress/integration/userCanChangeUILanguage.feature.js:11]: It didn't take it somehow, read a lot everywhere and all we do is right, we have to ask Thomas maybe.

**Pull request merged**